sqlglot/parser.py

```diff
@@ -1512,6 +1510,12 @@ def extend_props(temp_props: t.Optional[exp.Properties]) -> None:
                 schema=True, is_db_reference=create_token.token_type == TokenType.SCHEMA
             )
 
+            sequence_opts = (
+                self._parse_sequence_properties()
+                if create_token.token_type == TokenType.SEQUENCE
+                else None
+            )
+
```

Comment:
This isn't really what I had in mind - the idea was to handle any sequence properties inside of `_parse_property` so that they would be appended to `Create.args["properties"]` instead of in a new arg.